ecommerce_integrations/shopify/customer.py

```diff
@@ -81,25 +83,25 @@ def update_additional_address(
 			oldAddress.email_id = customer_mail
 			oldAddress.save()
 
-	def create_contact(self, first_name, last_name, customer_mail, phone, mobile, marketing) -> None:
+	def create_customer_contact(self, shopify_customer: Dict[str, Any]) -> None:
+
+		if not (shopify_customer.get("first_name") and shopify_customer.get("email")):
+			return
+
 		contact_fields = {
 			"status": "Passive",
-			"first_name": first_name,
-			"last_name": last_name,
-			"unsubscribed": not marketing,
+			"first_name": shopify_customer.get("first_name"),
+			"last_name": shopify_customer.get("last_name"),
+			"unsubscribed": shopify_customer.get("accepts_marketing"),
```

Comment:
ATTENTION this needs to be negated. There was a `not` statement before.

Reply:
Will fix it 👍